Replicator/RevFinder.hh

```diff
@@ -51,10 +51,14 @@ namespace litecore { namespace repl {
         /** Delegate must call this every time it receives a "rev" message. */
         void revReceived()     {enqueue(&RevFinder::_revReceived);}
 
+        /** Delegate calls this if it has to re-request a "rev" message, meaning that another call to
+            revReceived() will be made in the future. */
+        void reRequestingRev() {enqueue(&RevFinder::_reRequestingRev);}
+
     private:
         static const size_t kMaxPossibleAncestors = 10;
 
-        bool pullerHasCapacity() const   {return _pendingRevMessages <= tuning::kMaxPendingRevs;}
+        bool pullerHasCapacity() const   {return _numRevsBeingRequested <= tuning::kMaxPendingRevs;}
```

Comment:
Consider renaming the tuning variable to match the new variable name?

Reply:
Good idea. Done.